rezervo/providers/sats/urls.py

```diff
@@ -0,0 +1,14 @@
+BASE_PATH = "https://www.sats.no"
+
+SCHEDULE_PATH = "/api/group-exercise-search-day"
+LOGIN_PATH = "/logg-inn"
+AUTH_PATH = "/api/log-in"
+BOOK_PATH = "/api/book"
+CANCEL_BOOKING_PATH = "/api/unbook"
+BOOKINGS_PATH = "/min-side/kommende-trening"
+
+SCHEDULE_URL = BASE_PATH + SCHEDULE_PATH
+AUTH_URL = BASE_PATH + AUTH_PATH
+BOOKING_URL = BASE_PATH + BOOK_PATH
+UNBOOK_URL = BASE_PATH + CANCEL_BOOKING_PATH
```

Comment:
Rename this to `CANCEL_BOOKING_URL` and I'll stop complaining 🙃

Reply:
Haha no worries, that was just me going monkey brain when looked over the prior review comment. Feel free to scrutinize the code more if you'd like 😉 For instance, the fetcher for Sats proved to be a very interesting piece of aync programming 😛

Reply:
Fixed in https://github.com/mathiazom/rezervo/pull/43/commits/3ab507a6cd3f15d3ac4ead13ff6c6d8f0a5b3f29

Reply:
Yea yea, I only meant for this file :wink: